test/scripts/backup-restore/postgres.sh

```diff
@@ -1,235 +0,0 @@
-  aws rds restore-db-instance-from-db-snapshot \
-      --db-instance-identifier $AWS_DB_ID \
-      --db-snapshot-identifier $RDS_RESTORE_SNAPSHOT \
-      --db-subnet-group-name $DB_SUBNET_GROUP_NAME \
-      --vpc-security-group-ids $VPC_SECURITY_GROUP_IDS \
-      --multi-az \
-      --deletion-protection \
-      --region $AWS_REGION > /dev/null
-
-
-  # Wait for the database to be available
-  while true
-  do
-    STATUS=$(aws rds describe-db-instances --db-instance-identifier $AWS_DB_ID --region $AWS_REGION | jq -r '.DBInstances[0].DBInstanceStatus')
-    if [ "$STATUS" = 'available' ]; then
-      echo "Database restored."
-      break
-    fi
-
-    echo "Waiting for snapshot restoration... Current status: $STATUS"
-    sleep 10s
-  done
-
-  # Restore default scaling options that couldn't be added as part of the restore
-  echo "Restoring default scaling options..."
-  aws rds modify-db-instance \
-      --db-instance-identifier $AWS_DB_ID --max-allocated-storage 100 \
-      --region $AWS_REGION > /dev/null
-
-  # Revert PostGres CR Change
-  echo "Re-enabling automating RDS recreation..."
-  oc patch postgres/$POSTGRES_CR_NAME -n ${NS_PREFIX}-operator -p '{"spec":{"skipCreate":false}}' --type merge
-
-  # Dump the database after the restoration
-  echo "Dumping restored database..."
-  dump_database dump_after.sql
-  echo "Dumped restore database to dump_after.sql"
-
-  echo "Calculating difference between databases..."
-  DB_DIFF=$(diff dump_before.sql dump_after.sql)
-  if [ ! -z "$DB_DIFF" ]; then
-    echo "Difference found between database dumps:"
-    echo $DB_DIFF
-
-    exit 1
-  fi
-
-  echo "No difference between restored database."
-  echo "Deleting database dumps."
-  rm dump_before.sql
-  rm dump_after.sql
-
-  echo "Test finished successfully."
-}
-
-dump_database() {
-  DUMP_FILE=$1
-
-  echo "Creating throwaway Postgres container..."
-  cat << EOF | oc create -f - -n "$NS_PREFIX-operator"
-  apiVersion: integreatly.org/v1alpha1
-  kind: Postgres
-  metadata:
-    name: throw-away-postgres
-    labels:
-      productName: productName
-  spec:
-    secretRef:
-      name: throw-away-postgres-sec
-    tier: development
-    type: workshop
-EOF
-  
-  # Wait for the Postgres to be reconciled
-  while true
-  do
-      PHASE=$(oc get postgres/throw-away-postgres -n ${NS_PREFIX}-operator -o template --template={{.status.phase}})
-      if [ "$PHASE" = 'complete' ]; then
-        break
-      fi
-
-      echo "Waiting for throwaway Postgres container to complete. Current phase: $PHASE..."
-      sleep 10s
-  done
-
-  # Create the dump
-  kubectl exec deploy/throw-away-postgres \
-    -n ${NS_PREFIX}-operator \
-    -- env PGPASSWORD=$DB_PASSWORD pg_dump -h $DB_HOST -p $DB_PORT -U $DB_USER $DATABASE_NAME > $DUMP_FILE
-
-  # Delete the throwaway postgres
-  oc delete postgres/throw-away-postgres -n ${NS_PREFIX}-operator
-}
```

Comment:
This script is used as the subscript by the following backup & restore test cases, so proabably best to keep this :thinking: 
* https://github.com/integr8ly/integreatly-operator/tree/master/test-cases/tests/backup-restore